### PR TITLE
Always have children for confirm modal

### PR DESCRIPTION
**Pull request by kennyt**
If this doesn't get merged, this warning will show:

px

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @philipnrmn

File: src/js/components/ServicesTable.js [hunk 1]
```diff
@@ -219,26 +219,28 @@ var ServicesTable = React.createClass({
...
-          <p>
-            Are you sure you want to destroy <span className="emphasize">{service.getId()}</span>? This action is irreversible.
-          </p>
-          {this.getErrorMessage()}
-        </div>
-      );
+      serviceName = service.getId();
     }
 
+    let message = (
```
Comment (mlunoe) [inline src/js/components/ServicesTable.js]: If we are always going to have this as children, why not just put it as 'children'?
```js
    return (
      <Confirm
        ...>
        <div className="container-pod flush-top container-pod-short-bottom">
          <h2 className="text-danger text-align-center flush-top">Destroy {itemText}</h2>
          <p>
            Are you sure you want to destroy <span className="emphasize">{serviceName}</span>? This action is irreversible.
          </p>
          {this.getErrorMessage()}
        </div>
      </Confirm>
    );
```

**Comment by mlunoe** [comment]
Thanks for fixing this btw!

File: src/js/components/ServicesTable.js (hunk 1 above)
Comment (kennyt) [inline src/js/components/ServicesTable.js]: honestly you're right. i was just going the way it was being done at that point. i'll make the change right now

Comment (kennyt) [inline src/js/components/ServicesTable.js]: done

**Pull request merged**